owasm/Cargo.toml

```diff
@@ -23,4 +23,14 @@ members = [
     "chaintests/get_env",
     "chaintests/gold_price",
     "chaindebugtests/allocate",
+    "chaintests/alphavantage",
```

Comment:
Move all below chaintest/gold_price

Reply:
sorry for my confusing comment, I just want old older and new items come next gold_price and end with `chaindebugtest/allocate`